src/Microsoft.DotNet.ProjectModel/Files/PatternGroup.cs

```diff
@@ -12,6 +12,8 @@ namespace Microsoft.DotNet.ProjectModel.Files
 {
     public class PatternGroup
     {
+        private static readonly Dictionary<string, IEnumerable<string>> s_resolvedFilesCache = new Dictionary<string, IEnumerable<string>>();
```

Comment:
@troydai could you point me to this piece of code, so I can take a look at this?